po/es.po

```diff
@@ -15260,39 +15262,39 @@ msgstr "No hay ninguna cámara con soporte de captura disponible para su uso..."
 
 #: ../src/views/view.c:2143
 msgid "Left click"
-msgstr "Clic izquierdo"
+msgstr "Clic"
 
 #: ../src/views/view.c:2146
 msgid "Right click"
-msgstr "Clic derecho"
+msgstr "Clic Secundario"
 
 #: ../src/views/view.c:2149
 msgid "Middle click"
-msgstr "Clic medio"
+msgstr "Clic Medio"
 
 #: ../src/views/view.c:2152
 msgid "Scroll"
 msgstr "Desplazamiento"
 
 #: ../src/views/view.c:2155
 msgid "Left double-click"
-msgstr "Doble clic izquierdo"
+msgstr "Pulsación Doble"
 
 #: ../src/views/view.c:2158
 msgid "Right double-click"
-msgstr "Doble clic derecho"
+msgstr "Doble Clic Secundario"
 
 #: ../src/views/view.c:2161
 msgid "Drag and drop"
 msgstr "Arrastrar y Soltar"
 
 #: ../src/views/view.c:2164
 msgid "Left click+Drag"
-msgstr "Clic izquierdo + arrastrar"
+msgstr "Clic+Arrastre"
```

Comment:
I would leave it in infinitive form, with capital letter only at the beginning and white spaces: "Clic + arrastrar"

Reply:
It is more usual to have the key combinations without spaces unless there are compound words. And have the initial letter capitalized to specify that it is part of the key combination.
![imagen](https://user-images.githubusercontent.com/47407674/71482444-3d15c200-27c8-11ea-9f42-a0cd7c2e0e1f.png)

Reply:
Furthermore, adding spaces could probably be done with CSS know.